plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/SpanAddingClassVisitorFactory.kt

```diff
@@ -51,6 +32,29 @@ abstract class SpanAddingClassVisitorFactory :
         val debug: Property<Boolean>
 
         @get:Internal
-        val tmpDir: RegularFileProperty
+        val tmpDir: Property<File>
     }
+
+    companion object {
+        private val instrumentables: MutableList<ClassInstrumentable> = mutableListOf(
+            AndroidXSQLiteDatabase(),
+            AndroidXSQLiteStatement()
+//            AndroidXRoomDao()
```

Comment:
Was this intended?

Reply:
yep, Room instrumentation still wip, following PRs will complete it